index.bs

```diff
@@ -3747,25 +3746,25 @@ Record=] of type <code>throw</code>, |exception|, is given by:
 [=Remote end definition=] and [=local end definition=]
 
 <pre class="cddl local-cddl remote-cddl">
-ScriptResult = (
-  ScriptResultSuccess //
-  ScriptResultException
+ScriptEvaluateResult = (
```

Comment:
The section is called `#### The script.Result type #### {#type-script-Result}`. Should it be renamed to `#### The script.ScriptEvaluateResult type #### {#type-script-ScriptEvaluateResult}` ?